queries.js

```diff
@@ -0,0 +1,61 @@
+const pgp = require('pg-promise')()
+const connectionString = 'postgres://localhost:5432/sweethornbillbookstore'
```

Comment:
put this in an env file? does this work? shouldn't your user name be somewhere in this connection string?